### DISCUSSION TITLE
Email Automation

### DISCUSSION THREAD

**Discussion by danbennett123** [Budibase/budibase#3452] [Help] [answered]
Hi, loving budibase so far. I am in the process of building a light weight order management system. Its going well but i have an issue with the email automation. I get the exact response i want when i run test but when a new row is created via the usual running of the app the automation is not happening. 

The backend is a mysql databse if this makes any difference.

Any help would be much appreciated.

![Budibase-Email-Issue](https://user-images.githubusercontent.com/94695208/142635203-11e86007-b993-433c-acfc-471237d71657.jpg)

**Comment by danbennett123** [comment]
Soo not sure if this is intentional or not but it works when i go to apps - view published app. If i try it through the play symbol to launch dev version it doesnt work. The builds are identical and i confirmed this by making a slight tweak then publishing so both dev and production are the same. 

Maybe this is so automations don't come out of 1000 quota for budibase hosted apps?

**Comment by PClmnt** [comment, marked as answer]
Hi @danbennett123 , this is by design. You can see the rationale for this via the comments in this issue: https://github.com/Budibase/budibase/issues/2942

We plan on updating the UI to make this more obvious.

**Reply by danbennett123** [reply]
Hi, thanks for confirming this for me. I agree it would, especially for novice users, be helpful to be notified that this is the case in the automations section. Thankyou.
